packages/apple-pay-integration/project.json

```diff
@@ -0,0 +1,24 @@
+{
+  "root": "packages/apple-pay-integration",
+  "sourceRoot": "packages/apple-pay-integration/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/apple-pay-integration/**/*.{ts,tsx}"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": [
+        "coverage/packages/apple-pay-integration"
+      ],
+      "options": {
+        "jestConfig": "packages/apple-pay-integration/jest.config.js"
+      }
+    }
+  },
+  "tags": []
```

Comment:
Should we start introducing tags? Like `scope:integration`, `scope:core`, `scope:shared` etc... so we can impose [constraints](https://nx.dev/structure/monorepo-tags#imposing-constraints-on-the-project-graph) on what packages could be imported? cc @animesh1987

Reply:
Yes, I think it's the right time to do it now 😄

Reply:
On it.